ts/features/itwallet/issuance/screens/ItwIssuanceCredentialTrustIssuerScreen.tsx

```diff
@@ -89,17 +100,28 @@ type ContentViewProps = {
  * Renders the content of the screen
  */
 const ContentView = ({ credentialType, eid }: ContentViewProps) => {
+  const route = useRoute();
+  const mixPanelCredential = useMemo(
+    () => CREDENTIALS_MAP[credentialType as CredentialType],
+    [credentialType]
+  );
```

Comment:
Do we really need to memoize it? Can't we access directly from the function that uses it?
Also, `CredentialType` is a mock value, use a simple `string` instead